typescript/sdk/src/gas/token-prices.ts

```diff
@@ -136,6 +136,8 @@ export class CoinGeckoTokenPriceGetter implements TokenPriceGetter {
     const ids = chains.map(
       (chain) => this.metadata[chain].gasCurrencyCoinGeckoId || chain,
     );
+    // Coingecko rate limits, so we are adding this sleep
+    await sleep(5000);
```

Comment:
I believe this is causing the test to break due to timeout

Reply:
Fixed it by allowing configuration